### ISSUE TITLE
Failing to update portfolio and cash Balance

### ISSUE THREAD

**Issue by att-c** [krambox/moneymoney-ibkr#2]
After successfully setting up the account, I intermittently get the following error message when trying to update transactions and account balances: 
`ibkr.lua:76: attempt to index local 'positions' (a nil value)`

**Comment by 8986ca6c** [comment]
for me this sometimes also occurs. It seems to be only when I am updating the specific account (but it's not always an error then). Just update the account that is the parent of these two accounts. Then it works more reliable.
I named them

CapTrader FlexQuery
|---- IBKR Open Positions
|---- IBKR Cash Balance

So when I update CapTrader Flex Query, then everything works. Hope this helps

**Comment by krambox** [comment]
Yes, the Flex query must have the relevant fields as described in the readme. Currently I have only German screens. If someone has English screens, I will add the instructions.

**Comment by 8986ca6c** [comment]
My comment seems to be outdated. Can'T get it working anymore. I'll check the fields in the Felx Query later..

**Comment by 8986ca6c** [comment]
I have the same settings as stated in the Wiki. But there is one section wehre I dont't find instructions. See enclosed 
<img width="1170" alt="grafik" src="https://user-images.githubusercontent.com/30111613/155356385-83389668-50fb-49bc-8257-8dbd9f3dd76f.png">

These are the settings that worked earlier.  See comment below for the right settings.

**Comment by 8986ca6c** [comment]
If you choose "Optional", "XML" and "last Business day" (or similar, in German it's "Letzter Geschäftstag") it mmediately worked for me.

**Comment by hartmood** [comment]
My error in the MoneyMoney protocol is:
`           cash="xxx.2" cashLong="xxx.2" cashShort="0" 
          ibkr.lua:136: attempt to index local 'summary' (a nil value)
`
